app/scripts/modules/core/src/pipeline/config/stages/wait/WaitExecutionLabel.tsx

```diff
@@ -1,49 +1,43 @@
 import * as React from 'react';
 import { OverlayTrigger, Tooltip } from 'react-bootstrap';
 
-import { IExecutionStageSummary, IExecution } from 'core/domain';
-import { Application } from 'core/application/application.model';
+import { IExecutionStageLabelComponentProps } from 'core/domain';
 import { HoverablePopover } from 'core/presentation/HoverablePopover';
 import { ExecutionBarLabel } from 'core/pipeline/config/stages/common/ExecutionBarLabel';
-import { SkipWait } from './SkipWait';
 
-export interface IWaitExecutionLabelProps {
-  stage: IExecutionStageSummary;
-  execution: IExecution;
-  application: Application;
-  executionMarker: boolean;
-}
+import { SkipWait } from './SkipWait';
 
 export interface IWaitExecutionLabelState {
   target?: any;
 }
 
-export class WaitExecutionLabel extends React.Component<IWaitExecutionLabelProps, IWaitExecutionLabelState> {
-  constructor(props: IWaitExecutionLabelProps) {
+export class WaitExecutionLabel extends React.Component<IExecutionStageLabelComponentProps, IWaitExecutionLabelState> {
```

Comment:
While I agree that's a better name, the interface itself was already there (I just discovered and used it instead of the duplicate interface that was in this file before). I was hesitant to do a rename because two similar interfaces in [IExecutionStage.ts](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/domain/IExecutionStage.ts) have the same `ComponentXyz` naming.

Looking at the three similiarly-named interfaces, though, they're each only used in 2 or 3 spots so maybe a mass renaming is safe to do in this PR.

Reply:
do what you want

Reply:
I WILL do what I want https://github.com/spinnaker/deck/pull/6826/commits/8985988b9d3fd69cdad4ae5995b532df31df1ea4